code/src/styles/MovieStyle.css

```diff
@@ -0,0 +1,359 @@
+/***GENERAL***/
+* {
+	margin: 0;
+	padding: 0;
+	box-sizing: border-box;
+}
+
+h1 {
+	font-size: 24px;
+}
+
+h2 {
+	margin-bottom: 10px;
+	font-size: 22px;
+}
+
+h3,
+h4 {
+	margin-bottom: 10px;
+}
+
+a {
+	color: #fff;
+	text-decoration: none;
+}
+
+/***NAV***/
+header {
+	height: 10vh;
+	display: flex;
+	justify-content: space-between;
+	align-items: center;
+	padding: 20px;
+	position: relative;
+}
+
+nav {
+	display: flex;
+	justify-content: space-between;
+	align-items: center;
+	width: 100%;
+}
+
+.nav__button {
+	display: inline;
+	border: none;
+	background: transparent;
+	color: #fff;
+	font-size: 16px;
+	padding: 10px;
+	cursor: pointer;
+	font-size: 25px;
+}
+
+.nav__button:hover {
+	background: #1f6f8b;
+}
+
+.nav__button--open {
+	display: flex;
+	flex-direction: column;
+	align-items: center;
+	justify-content: space-evenly;
+	position: absolute;
+	top: 10vh;
+	left: 0;
+	right: 0;
+	z-index: 1;
+	background: #000;
+	height: 25vh;
+}
+
+.nav__button--closed {
+	display: none;
+}
+
+.nav__link {
+	padding: 10px;
+	text-align: center;
+}
+
+.nav__link:hover {
+	background: #1f6f8b;
+}
+
+/*-COMPANY HEADER-*/
+.company-name {
+	height: 100%;
+}
+
+.company__logo {
+	height: 100%;
+	filter: invert(1);
+	color: #000;
+}
+
+/***MOVIELIST***/
+.list {
+	display: grid;
+	grid-template-columns: 1fr 1fr;
+	width: 100%;
+}
+
+/*-MOVIE THUMB-*/
+.movie-thumb {
+	position: relative;
+}
+
+.movie-thumb__img {
+	width: 100%;
+}
+
+.movie-thumb:hover .movie-thumb__info {
+	background: rgba(0, 0, 0, 0.7);
+}
+
+.movie-thumb__info {
+	position: absolute;
+	top: 0;
+	bottom: 0;
+	left: 0;
+	right: 0;
+	display: none;
+	transition: all 0.5s ease-in-out;
+}
+
+.movie-thumb:hover .movie-thumb__info {
+	display: flex;
+	flex-direction: column;
+	justify-content: flex-end;
+	padding: 40px 10px;
+}
+
+/***MOVIEPAGE***/
+/*-MOVIE DETAIL-*/
+.details {
+	background-size: cover;
+	background-repeat: 'no-repeat';
+	min-height: 100vh;
+	background-position: center;
+	background-position-x: center;
+	background-position-y: center;
+	padding: 20px;
+	display: flex;
+	flex-direction: column;
+	justify-content: flex-end;
+}
+
+.details-container {
+	display: grid;
+	grid-template-columns: repeat(3, 1fr);
+	grid-column-gap: 20px;
+	max-width: 1000px;
+	background: rgba(0, 0, 0, 0.4);
+}
```

Comment:
I noticed that on small mobile screens like iPhone 5 the details container covers the back-button on some of the movies that has a longer text. A solution to this might be to add like margin-top: 10vh; or something similar here to avoid that.